model/project_ref.go

```diff
@@ -1396,6 +1396,14 @@ func FindMergedProjectRefsByIds(ids ...string) ([]ProjectRef, error) {
 	}, true)
 }
 
+func FindProjectsByIdsWithLoggerAndRepoSettings(ids ...string) ([]ProjectRef, error) {
```

Comment:
Should also remove this function, (it can be helpful to give a PR a proof read before re-requesting review to catch some of these smaller things)

Reply:
oops thanks for catching!